core/src/main/java/io/grpc/inprocess/InProcessServer.java

```diff
@@ -55,13 +52,6 @@ static InProcessServer findServer(String name) {
    */
```

Comment:
You need to update schedulerPool's documentation.

Reply:
Fixed